bin/node/runtime/src/lib.rs

```diff
@@ -1626,4 +1626,14 @@ mod tests {
 
 		is_submit_signed_transaction::<Runtime>();
 	}
+
+	#[test]
+	fn call_size() {
+		assert!(
+			core::mem::size_of::<Call>() <= 200,
```

Comment:
200 is an arbitrary number?

Reply:
yes IIRC it is the same number as clippy lint

Reply:
what clippy lint?

Reply:
this one https://rust-lang.github.io/rust-clippy/master/#large_enum_variant